config/samples/control_v1alpha1_placement.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: control.kubestellar.io/v1alpha1
-kind: Placement
+kind: BindingPolicy
 metadata:
-  name: placement1
+  name: bindingpolicy1
 spec:
   clusterSelectors:
   - matchLabels: {"location-group":"edge"}
   downsync:
-  - labelSelectors:
+  - objectSelectors:
```

Comment:
I see that this change is not due to the renaming in this PR but an API change made earlier. Since this inconsistency went undetected since then I wonder: is this file even being used? Should we delete it? If not then there is a test or procedure that needs to be enhanced so that this bug would have been detected.

Reply:
I asked myself the same question and recorded the question in the history in this [commit](https://github.com/kubestellar/kubestellar/commit/0510ca35d808cebee3d5bfed7fe1c3f47d559abc). I thought someone created these files for their continence of quick tests, didn't want to create any surprise to them so I kept them and updated them.

Today I traced back to the very beginning of the history and realized that 'someone' is actually myself. With that I can safely delete them now.